downstream/modules/hub/proc-obtain-images.adoc

```diff
@@ -43,4 +58,6 @@ $ podman images
 [role="_additional-resources"]
 .Additional resources
 
-* See link:https://redhat-connect.gitbook.io/catalog-help/[Red Hat Ecosystem Catalog Help] for information on registering and getting images.
+* See link:redhat-connect.gitbook.io/catalog-help/[Red Hat Ecosystem Catalog Help] for information on registering and getting images.
+
+* See link:{BaseURL}subscription_central/1-latest/html/creating_and_managing_manifests_for_a_connected_satellite_server/index[Creating and managing manifests for a connected Satellite Server] to learn more about the changes coming to Red Hat subscription tooling
```

Comment:
I believe there needs to be a slash right after the base URL attribute for this to work--so it should be 
{BaseURL}/subscription_central/1-latest/html/creating_and_managing_manifests_for_a_connected_satellite_server/index